### PR TITLE
add ;tripmine command // Update Fun.lua

### PR DESCRIPTION
**Pull request by KittyTheDeveloper**
# ;tripmine // give subspace tripmine
it's self explanatory, it gives a subspace tripmine to the player.

### CODE REVIEW

File: MainModule/Server/Commands/Fun.lua [hunk 1]
```diff
@@ -1092,7 +1092,23 @@ return function(Vargs, env)
...
-
+		Tripmine = { --// D I S A S T E R
+			Prefix = Settings.Prefix;
+			Commands = {"tripmine", "subspacetripmine","subspace"};
+			Args = {"player"};
+			Description = "Gives the target player(s) a Subspace Tripmine";
+			AdminLevel = "Admins";
+			Fun = true;
+			Function = function(plr: Player, args: {string})
+				local subspace = service.Insert(11999247)
```
Comment (SuperCater) [inline MainModule/Server/Commands/Fun.lua:1103]: Shouldn't this be deleted once the command is finished? I'm not sure if the GC deletes instances that are unused

Comment (KittyTheDeveloper) [reply MainModule/Server/Commands/Fun.lua:1103]: could be.

**Comment by kaiserandaxl** [comment]
Sorry but you can't use the insert because the player doesn't own the asset

**Comment by PurpleCreativity** [comment]
dont make smth a command if it can be made easily into an alias

**Review by PurpleCreativity — CHANGES_REQUESTED** [review]
no

**Comment by kaiserandaxl** [comment]
This is a dumb command tho, why do you need a command that gives somebody a bomb or trip mine. Just use the gear command.

**Review by WalkerOfBacon — COMMENTED** [review]
Why are you inserting it when the item itself is a gear

**Comment by Dimenpsyonal** [comment]
> dont make smth a command if it can be made easily into an alias


my guy has forgotten the :sword command !!

**Comment by ccuser44** [comment]
> > dont make smth a command if it can be made easily into an alias
> 
> my guy has forgotten the :sword command !!

Its for a special reason
The same doesnt apply to tripmines

**Comment by Dimenpsyonal** [comment]
I don't think this is necessary, :sword is there because lots of games use it for fighting whereas the subspace mine is only really used as a prank or in obstacle courses

**Pull request closed without merging**